spec/ParseAPI.spec.js

```diff
@@ -137,7 +162,12 @@ describe('miscellaneous', function() {
     // Create a new server to try to recreate the unique indexes
     .then(reconfigureServer)
     .catch(error => {
-      expect(error.code).toEqual(Parse.Error.DUPLICATE_VALUE);
+      on_db('mongo', () => {
+        expect(error.code).toEqual(Parse.Error.DUPLICATE_VALUE);
+      });
+      on_db('postgres', () => {
+        expect(error.code).toEqual('23505');
```

Comment:
I think this test should still be excluded in PG if it's not returning the right error code in PG. I thought `on_db` was going to be reserved for unsupported operations (like PG has no equivalent of mongo's $pullAll)